llvm/test/CodeGen/Generic/allow-check.ll

```diff
@@ -1,5 +1,5 @@
 ; Avoid `!DL->isLittleEndian() && !CLI->enableBigEndian()` missmatch on PPC64BE.
```

Comment:
Is this comment still appropriate?

Reply:
Really, what we want to check here is "does the target support globalisel", since we only want to check the behavior of `-global-isel=1` on such targets.  But the comment is basically correct; big-endian targets don't support globalisel (with the exception of m68k), so checking endianness is a reasonable approximation.